### PR TITLE
Calculate CC and compound names

### PR DESCRIPTION
**Pull request by ghost**
Calculate extended cyclomatic complexity and number of compound name for **local** variables inside methods

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -51,7 +53,7 @@ uncalc:
 
 calc:
 	mkdir -p metrics
-	for r in $$(find clones/ -type directory -depth 2); do \
+	for r in $$(find clones/ -type d -maxdepth 2 ); do \
```
Comment (ghost) [inline Makefile:60]: For compatibility run on ubuntu

File: computation/calc.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    return count
+
+"""Check the name for compound inside the methods (i.e. for local variables)
+
+:param node: class provided by the parser and targeted to Java 8 spec
+:returns: boolean
+"""
+def compound(node):
+    flag = False
+    if (isinstance(node, tree.LocalVariableDeclaration)):
```
Comment (ghost) [inline computation/calc.py:60]: calculate compound names for **local** variables

**Comment by yegor256** [comment]
@goonnors how do I run the tests? Maybe we should add `make test` target to the `Makefile`?

**Comment by ghost** [comment]
I added `calc_test` target to the `Makefile` to run the tests

**Comment by yegor256** [comment]
@goonnors thanks!

**Pull request merged**